Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -128,6 +128,12 @@ extension TuistGraph.Target {
                 return []
             }
         }
+        
+        // remove excluding
+        manifest.resources?.excluding.forEach { path in
+            allResources.remove(path: AbsolutePath(path.pathString))
+        }
```

Comment:
This is not taking globs into account, is it?
I think we should add here the same logic we have for excluding from sources

Reply:
now it does